ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/IntTopkDistinctKudafTest.java

```diff
@@ -118,22 +118,21 @@ public void shouldBeThreadSafe() {
     final List<Integer> values = ImmutableList.of(10, 30, 45, 10, 50, 60, 20, 70, 80, 35, 25);
 
     // When:
-    final Object[] result = IntStream.range(0, 4)
+    final List<Integer> result = IntStream.range(0, 4)
         .parallel()
         .mapToObj(threadNum -> {
-          Integer[] aggregate = new Integer[]
-              {1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
+          List<Integer> aggregate = ImmutableList.of(1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1);
```

Comment:
Would think this needs to be a mutable List as its passed to the `aggregate` function, which mutates its input. e.g.

```
 List<Integer> aggregate = new ArrayList<>(ImmutableList.of(1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1));
```

Reply:
Yes, It's now mutable.